content/v2.0/reference/glossary.md

```diff
@@ -9,7 +9,12 @@ menu:
 v2.0/tags: [glossary]
 ---
 
-[A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g) | [H](#h) | [I](#i) | [J](#j) | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o) | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [X](#x) | [Y](#y) | [Z](#z)
+|         |         |         |         |         |         |         |         |
+| :--     | :--     | :--     | :--     | :--     | :--     | :--     | :--     |
+| [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g) | [H](#h) |
+| [I](#i) | [J](#j) | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o) | [P](#p) |
+| [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | X       |
+| Y       | Z       |         |         |         |         |         |         |
```

Comment:
Let's not format this as a table. This information doesn't really make sense in a table and it won't scale well for mobile. The previous structure just wraps on line endings and scales better.

Reply:
Okay. Will kill this commit—